src/onboarding/github.md

```diff
@@ -0,0 +1,61 @@
+# GitHub Tickets
+
+GitHub is a service that provides code hosting and versioning. We also use it for its collaborative features, such as issue/bug tracking and project boards. This documents explains how to use GitHub to keep track of tasks.
+
+## Repositories
+
+The main repository for the Toronto Community Network can be found on GitHub under [tomeshnet/toronto-community-network](https://github.com/tomeshnet/toronto-community-network).
+
+![homepage](./images/homepage.png)
+
+A repository (or "repo") is a collection of files, issues, and project boards grouped under a name. The name of this one is `toronto-community-network` and that it was created by the [tomeshnet](https://github.com/tomeshnet) organization.
+
+An overview of the files and folders is displayed in the middle of the screen, and below that the README is shown, which is a file that explains what the repository is for. If you click on `README.md` you'll see it has the same contents as the text on the homepage.
+
+## Issues
+
+If you click on `Issues` tab you'll see this:
+
+![issues](./images/issues.png)
+
+This tab provides a list of all the issues that have been created for this repository. Issues are similar to the idea of "tickets" or bug reports. It allows us to keep track of tasks or potential plans. All issues have numbers, referenced with a hashtag, such as #32, which can be found [here](https://github.com/tomeshnet/toronto-community-network/issues/32).
```

Comment:
"allows us" tone

Also hashtag notation - actually gitHub Autolinked References
https://docs.github.com/en/github/writing-on-github/autolinked-references-and-urls